cloudevents/sdk/http_events.py

```diff
@@ -64,8 +65,27 @@ def from_http(
         if data_unmarshaller is None:
             data_unmarshaller = _json_or_string
 
-        event = marshaller.NewDefaultHTTPMarshaller().FromRequest(
-            v1.Event(), headers, data, data_unmarshaller
+        headers = {key.lower(): value for key, value in headers.items()}
+
+        content_type = headers.get("content-type", None)
+
+        specversion = "1.0"
+        marshall = marshaller.NewDefaultHTTPMarshaller()
+
+        for cnvrtr in marshall.http_converters:
```

Comment:
I understand you didn't introduce this change, but can you use full words `converter`?

Reply:
LOL yes of course!